.env

```diff
@@ -4,9 +4,16 @@ POSTGRES_PASSWORD=polkastats
 POSTGRES_DATABASE=polkastats
 POSTGRES_HOST=localhost
 POSTGRES_PORT=5432
-LOGGING=1
-WS_PROVIDER_URL=wss://ws-rc.unique.network
-TYPE_PROVIDER=quartz
+LOGGING=false
```

Comment:
In TypeORM config we waiting number for define logging level

https://github.com/UniqueNetwork/unique-scan-backend/pull/40/files#diff-9af894d792453ec3d2a15d629d9731b3cd7d17d7de45b7f0cf64b671d9246c15R42

Maybe better to use enum with names of logging levels like - info, debug, warning, error. But it's not the task for this PR.

Reply:
As they say "moped is not mine")) Ok, I will change value to 0.